src/tests/activities.test.js

```diff
@@ -1,3 +1,130 @@
-test('should pass a basic test', () => {
-  expect(1 + 1).toBe(2);
+import request from 'supertest';
+import app from '../app';
+import activity from '../models/Activity';
+import activitySeed from '../seeds/activities';
+
+const activities = {
+  name: 'voley',
+  description: 'basic training that consists of throwing soft punches into the air with ertert erte reter dfd er ert  ert',
+  isActive: true,
+};
+
+const ActivityBad = {
+  name: '',
+  description: '',
+  isActive: true,
+};
+
+const updatedActivity = {
+  name: 'basquetball',
+  description: 'This is the most practiced sport nowadays by teams of men and women, both professionally and amateurishly.',
+  isActive: false,
+};
+
+beforeAll(async () => {
+  await activity.collection.insertMany(activitySeed);
+});
+
+describe('POST /api/activities', () => {
+  test('should create a activity', async () => {
+    const res = await request(app).post('/api/activities').send(activities);
+    expect(res.status).toBe(201);
+    expect(res.body).toBeDefined();
+    expect(res.error).toBeFalsy();
+  });
+
+  test('should return a error', async () => {
+    const res = await request(app).post('/api/activities').send(ActivityBad);
+    expect(res.status).toBe(400);
+    expect(res.body.data).toBeUndefined();
+    expect(res.error).toBeTruthy();
+  });
+});
+
+describe('PUT /api/activities/:id', () => {
```

Comment:
Maybe you could test passing the same information, and it should return that there were no changes